location_tag.md

```diff
@@ -0,0 +1,110 @@
+# Location Tag Type
+
+## Overview
+
+* Tags are strings attached to an entity (like tagging of blog entries) to form a folksonomy and/or to categorize data. 
+
+* Location Tags are being used to tag locations.Which enables the system to act based on the presence of tags on certain locations. 
+
+* You should not use locations to represent logical ideas like All District Hospitals they should be modeled using LocationTags.
+
+## Available operations. 
+
+1. [List location tags](#list-location-tags)
+2. [Create a location tag](#create-a-location-tag)
+3. [Update a location tag](#update-a-location-tag)
+4. [Delete a location tag](#delete-a-location-tag)
+
+
+### List location tags
+
+* #### List all non-retired location tags.
+
+    Quickly filter location tags with a given search query. Returns a `404 Not Found` status if the location tag not exists.
+    If the user not logged in to  perform this action, a `401 Unauthorized` status returned.
+
+    ##### Query Parameters
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *q* | `Search Query` | Display Name of Location location tag type.
+
+    ```console
+    GET /locationtag?q="visit"
+     ```
+
+* #### List location tag by UUID.
+
+    Retrieve a location tag by its UUID. Returns a `404 Not Found` status if the location tag type not exists. If the 
+    user not logged in to  perform this action, a `401 Unauthorized` status returned.
+
+    ```console
+    GET /locationtag/:target_location_tag_uuid
+    ```
+
+### Create a location tag
+
+* To Create a location tag  you need to specify below attributes in the request body.If the user not logged in to perform this action,
+ a `401 Unauthorized` status returned.
+
+    #### Attributes
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *name* | `String` | Name of the location tag  (Required)
+    *description* | `String` | Description (Required)
+    *retired* | `Boolean` | Retired status of the location tag
+    *retiredReason* | `String` | Reason to retire the location tag
```

Comment:
For location tags that are retired, this may contain an explanation of why the location tag was retired.